### PR TITLE
pwa about page

### PR DESCRIPTION
**Pull request by shuklabhay**
about page

## Description
about page

## Motivation and Context
pwa about page
[
](https://github.com/the-blue-alliance/the-blue-alliance/issues/6164)

## How Has This Been Tested?
local run

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

File: pwa/app/routes/about.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            Since 2022, The Blue Alliance has been a 501(c)(3) non-profit
+            organization. We rely on community donations to pay for hosting
+            costs.
+          </p>
+
+          <p>
+            You can support The Blue Alliance or reach us with the following:
+          </p>
+          <Button
+            onClick={() =>
```
Comment (fangeugene) [inline pwa/app/routes/about.tsx:30]: Shouldn't manually be setting the window location. External sites should just be a normal `<a>` that's formatted to look like a button.

File: pwa/app/routes/about.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+              (location.href = 'https://github.com/the-blue-alliance')
+            }
+          >
+            Contribute on GitHub!
+          </Button>
+          <Button onClick={() => (window.location.href = '/donate')}>
```
Comment (fangeugene) [inline pwa/app/routes/about.tsx:36]: Same as above, but with `<Link>`

**Review by fangeugene — CHANGES_REQUESTED** [review]
Added some comments

File: pwa/app/routes/about.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+          <p>
+            Since 2022, The Blue Alliance has been a 501(c)(3) non-profit
+            organization. We rely on community donations to pay for hosting
+            costs.
+          </p>
+
+          <p>
+            You can support The Blue Alliance or reach us with the following:
+          </p>
+          <a
```
Comment (fangeugene) [inline pwa/app/routes/about.tsx:31]:
```
<Button asChild>
  <a href="https://github.com/the-blue-alliance">
    Contribute on GitHub!
  </a>
</Button>
```

File: pwa/app/routes/about.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+              <p>
+                <a href="https://www.chiefdelphi.com/" title="Chief Delphi">
+                  Chief Delphi
+                </a>{' '}
+                - The go-to fourm for FRC discussion
+              </p>
+            </li>
+            <li className="list-disc">
+              <p>
+                <a href="http://frc.link/" title="FRC Links">
```
Comment (fangeugene) [inline pwa/app/routes/about.tsx:128]: https

File: pwa/app/routes/about.tsx [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+            </li>
+            <li className="list-disc">
+              <p>
+                <a href="https://www.statbotics.io/" title="Statbotics">
+                  Statbotics
+                </a>{' '}
+                - Unique FRC team and event performance comparisons
+              </p>
+            </li>
+            <li className="list-disc">
```
Comment (fangeugene) [inline pwa/app/routes/about.tsx:142]: Remove this

**Review by fangeugene — CHANGES_REQUESTED** [review]

File: pwa/app/routes/about.tsx [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+export default function About(): JSX.Element {
+  return (
+    <>
+      <div className="flex flex-col divide-y [&_*]:mt-4">
+        <div>
+          <h1 className="text-3xl font-medium">About Us</h1>
+          <p>
+            Founded in the fall of 2006, The Blue Alliance began as a website
+            dedicated to providing everyone involved in the FIRST Robotics
```
Comment (ofekashery) [inline pwa/app/routes/about.tsx:13]:
```suggestion
            dedicated to providing everyone involved in the <em>FIRST</em> Robotics
```

File: pwa/app/routes/about.tsx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+            Competition (FRC) with scouting data and match videos. Since then,
+            the project has grown to include developers from within the FIRST
```
Comment (ofekashery) [inline pwa/app/routes/about.tsx:15]:
```suggestion
            the project has grown to include developers from within the <em>FIRST</em>
```

File: pwa/app/routes/about.tsx [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+          </Link>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">
+            About <em>FIRST</em> and <em>FIRST</em> Robotics Competition
+          </h3>
+          <p>
+            <a href="http://www.firstinspires.org/">
+              {' '}
```
Comment (ofekashery) [inline pwa/app/routes/about.tsx:63]: Remove this.

File: pwa/app/routes/about.tsx [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+            gives high school students and their adult mentors the opportunity
+            to work and create together to solve a common problem. Each{' '}
+            <em>FIRST</em> Robotics Competition season culminates with local and
+            regional events where qualifying teams compete for awards and a spot
+            in the <em>FIRST</em> Championship.
+          </p>
+
+          <div className="flex">
+            <img
+              src="https://www.firstinspires.org/sites/default/files/uploads/resource_library/brand/thumbnails/FIRST-V.png"
```
Comment (ofekashery) [inline pwa/app/routes/about.tsx:86]: I think it’s better to move this to our assets directory.

File: pwa/app/routes/about.tsx (hunk 8 above)
Comment (shuklabhay) [reply pwa/app/routes/about.tsx:63]: sorry, what exactly should be removed?

File: pwa/app/routes/about.tsx (hunk 9 above)
Comment (shuklabhay) [reply pwa/app/routes/about.tsx:86]: are you referring to `/pwa/public`

File: pwa/app/routes/about.tsx (hunk 8 above)
Comment (fangeugene) [reply pwa/app/routes/about.tsx:52]: I think the`{' '}` above is unnecessary

**Review by fangeugene — APPROVED** [review]

**Pull request merged**